### PR TITLE
Set-up Portfolio's Project & About section

### PR DESCRIPTION
**Pull request by dhirajsingha143**
- Added Project section and,
- Added About section in Portfolio.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -25,5 +25,107 @@ <h1 class="headline-title">I'm Dhiraj<br>Glad to see<br>you!</h1>
             <li><a href="#"><img src="images/angellist.png" alt="Angellist"></a></li>
         </ul>
     </section>
+    <section class="project">
+        <div class="card">
```
Comment (joskalenda) [inline index.html:29]: - [ ] - [ OPTIONAL ] Well done on creating the readme file, but at this step, we agree that it should have all the necessary information about the project, what do you think if you make a few touches by adding the project images, description, title and remove the microverse logo, please

- [ ] Your code is well structured referring to the project requirements, congratulations @dhirajsingha143, but we have a few things to modify before the project is approved, let's start with the design below, your design is well done but we are missing some space as indicate the red marks compared to the original design, please make use you adjust them. 💯 
lang

- [ ] Take a look at this record below.

lang

- [ ] - [ OPTIONAL] I think we see the same thing right? starting with the logo name, you can take it as optional BUT I would suggest you change that text into something relevant for you as your name, in addition to that, the position property you gave it is causing a few problems when we scroll, there are 2ways you can fix this, one is by providing a background to the nav bar with the same exact color as you `section` with class `headline` and the second is to remove the fixed position so when we scroll, the nav bar goes with the whole page, I let you choose one 😄   

- [ ] You have indeed remarked on the blank spaces on the right side of your page, right? your sections are not well-adjusted in the middle of the page, please fix that too

**Review by joskalenda — CHANGES_REQUESTED** [review]
| STATUS: REQUIRED CHANGES ♻️  |
|--------------------------------|
| Hi @dhirajsingha143 , |
|Good job so far! There are some issues that you still need to work on to go to the next project but you are almost there! |
|  |

## Highlights

- [ ]...Great use of microverse readme template✅ 
- [ ]...Green linters check✅ 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

-------
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me(@joskalenda) in your question so I can receive the notification.

> **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: styles.css [hunk 2]
```diff
@@ -140,7 +135,317 @@ header a {
...
+  font-size: 12px;
+  height: 24px;
+  width: auto;
+  padding: 4px 12px 4px 12px;
+  border-radius: 8px;
+  color: #6070ff;
+  background-color: #ebebff;
+  display: flex;
+  align-self: center;
+}
```
Comment (Tufahel) [inline styles.css:300]: ***[OPTIONAL]*** 

- [ ] In [Figma Design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122&t=99Sj0XRcegz7sbHC-0) inside the card `html`, `css`, `javascript`, etc elements are centered inside the `button` or `li`. Kindly fix that.

- `Your Design`
Screenshot from 2022-12-03 16-25-57

- `Given Figma Design`
hello

**Review by Tufahel — APPROVED** [review]
# Status: Approved :heavy_check_mark: :heavy_check_mark: :heavy_check_mark:

Hi @dhirajsingha143  

Great work with UI. It looks awesome. :100: 
You have implemented all of the required changes. :+1:  Your project is complete now! There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To Highlight 👏🏻



:heavy_check_mark: No Linter Errors.
:heavy_check_mark: Correct GitHub Flow.
:heavy_check_mark: Descriptive PR and Commit Messages.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**